_wikis.json

```diff
@@ -466,6 +466,13 @@
         ],
         "setOnly": []
     },
+    {
+        "key": "wp",
+        "name":"Wikipedia"
```

Comment:
Missing a comma at the end of the line. Perhaps also clarify as Wikipedia (English)?